step-api/step-api-keyword/src/main/java/step/handlers/javahandler/AbstractKeyword.java

```diff
@@ -89,5 +89,18 @@ public void setProperties(Map<String, String> properties) {
 	public boolean onError(Exception e) {
 		return true;
 	}
+  
+	/**
+	 * Hook called before each keyword call.
+	 *
+	 * @param keyword: the keyword to be called
+	 */
+	public void beforeKeyword(Method keyword) {}

```

Comment:
@jeromebrongniart Any particular reason for passing the Method object instead of the Keyword annotation directly?

Reply:
No particular reason. I aggree that passing the annotation can be more meaningfull.
I will do the change

Reply:
Changed in https://github.com/exense/step-api/pull/18/commits/c878e4ee0690404a68dc2410a2e4926e2c263620